src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/ApplicationTimelineTransformerTest.kt

```diff
@@ -106,6 +110,47 @@ class ApplicationTimelineTransformerTest {
     )
   }
 
+  @ParameterizedTest
+  @MethodSource("domainEventTypeArgs")
+  fun `transformDomainEventSummaryToTimelineEvent adds appealUrl if the type is APPROVED_PREMISES_ASSESSMENT_APPEALED`(args: Pair<DomainEventType, TimelineEventType>) {
+    val applicationId = UUID.randomUUID()
+    val appealId = UUID.randomUUID()
+    val (domainEventType, timelineEventType) = args
+    val domainEvent = DomainEventSummaryImpl(
+      id = UUID.randomUUID().toString(),
+      type = domainEventType,
+      occurredAt = OffsetDateTime.now().toTimestamp(),
+      bookingId = null,
+      applicationId = applicationId,
+      assessmentId = null,
+      premisesId = null,
+      appealId = appealId,
+    )
+
+    every { mockDomainEventDescriber.getDescription(domainEvent) } returns "Some event"
+
+    val associatedUrls = if (domainEventType == DomainEventType.APPROVED_PREMISES_ASSESSMENT_APPEALED) {
+      listOf(
+        TimelineEventAssociatedUrl(TimelineEventUrlType.application, "http://somehost:3000/applications/$applicationId"),
```

Comment:
In our review session yesterday I think we agreed that this (assessment appealed timeline event) will be clearer with just **one** url - that of the appeal.

Reply:
I guess the question here would be - if someone is not an appeals manager, should they see no URLs at all against that event type?

Reply:
Ignore that, as above, I think we'll remove that restriction